Sample Project/TestBed/MasterViewController.m

```diff
@@ -0,0 +1,915 @@
+@property (strong, nonatomic) IBOutlet UISwitch *enableCustomViewSwitch;
+@property (strong, nonatomic) IBOutlet UITextField *noDataLabelTextField;
+@property (strong, nonatomic) IBOutlet UISwitch *enableNoDataLabelSwitch;
+
+@property (strong, nonatomic) IBOutlet UIButton *animationGraphStyleButton;
+@property (strong, nonatomic) IBOutlet UITextField *animationEntranceTime;
+@property (strong, nonatomic) IBOutlet UISwitch *dotsWhileAnimateSwitch;
+@property (strong, nonatomic) IBOutlet UISwitch *touchReportSwitch;
+@property (strong, nonatomic) IBOutlet UITextField *widthTouchInputLineField;
+
+@property (strong, nonatomic) IBOutlet UIButton *fontNameButton;
+@property (strong, nonatomic) IBOutlet UITextField *fontSizeField;
+@property (strong, nonatomic) IBOutlet UITextField *numberFormatField;
+
+@property (strong, nonatomic) IBOutlet UIView *colorTopChip;
+@property (strong, nonatomic) IBOutlet UISwitch *gradientTopSwitch;
+@property (strong, nonatomic) IBOutlet UIView *colorBottomChip;
+@property (strong, nonatomic) IBOutlet UISwitch *gradientBottomSwitch;
+@property (strong, nonatomic) IBOutlet UIView *colorLineChip;
+@property (strong, nonatomic) IBOutlet UIView *colorPointChip;
+@property (strong, nonatomic) IBOutlet UIView *colorTouchInputLineChip;
+@property (strong, nonatomic) IBOutlet UIView *colorXaxisLabelChip;
+@property (strong, nonatomic) IBOutlet UIView *colorBackgroundXaxisChip;
+@property (strong, nonatomic) IBOutlet UIView *colorYaxisLabelChip;
+@property (strong, nonatomic) IBOutlet UIView *colorBackgroundYaxisChip;
+@property (strong, nonatomic) IBOutlet UIView *colorBackgroundPopUpLabelChip;
+@property (strong, nonatomic) IBOutlet UISwitch *gradientLineSwitch;
+@property (strong, nonatomic) IBOutlet UISwitch *gradientHorizSwitch;
+
+@property (strong, nonatomic) IBOutlet UITextField *alphaTopField;
+@property (strong, nonatomic) IBOutlet UITextField *alphaBottomField;
+@property (strong, nonatomic) IBOutlet UITextField *alphaLineField;
+@property (strong, nonatomic) IBOutlet UITextField *alphaTouchInputLineField;
+@property (strong, nonatomic) IBOutlet UITextField *alphaBackgroundXaxisField;
+@property (strong, nonatomic) IBOutlet UITextField *alphaBackgroundYaxisField;
+
+@end
+
+@implementation MasterViewController
+
+static NSString * enableTouchReport = @"enableTouchReport";
+static NSString * lineChartPrefix = @"lineChart";
+
+CGGradientRef createGradient () {
+    CGColorSpaceRef colorspace = CGColorSpaceCreateDeviceRGB();
+    size_t num_locations = 2;
+    CGFloat locations[2] = { 0.0, 1.0 };
+    CGFloat components[8] = {
+        1.0, 1.0, 1.0, 1.0,
+        1.0, 1.0, 1.0, 0.0
+    };
+    CGGradientRef result =  CGGradientCreateWithColorComponents(colorspace, components, locations, num_locations);
+    CGColorSpaceRelease(colorspace);
+    return result;
+}
+
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view, typically from a nib.
```

Comment:
Let's set the title of the view controller somewhere in viewDidLoad: self.title = @"Options".
This way the back arrow will display "Options" instead of "Master" which doesn't really make sense.